### ISSUE TITLE
Handle entry point

### ISSUE THREAD

**Issue by ehsannas** [google/DirectXShaderCompiler#39]
**Goals**
* Emit the correct `OpEntryPoint` for `void main() {}` as pixel shader

**Comment by ehsannas** [comment]
pull #41 adds `OpEntryPoint` with the correct ExecutionModel, function ID and function name.

Keeping this issue open since Interface\<id\>s are not yet added to the OpEntryPoint instruction.

**Comment by antiagainst** [comment]
@ehsannas This can be marked as done once #41 is merged since it only asks the entry point for `void main() {}`. We can have another task for emitting interface ids later.

**Comment by antiagainst** [comment]
Design?: N/A
Pull request: #41 
Test coverage: #41 
Verifer: @antiagainst
